seqr/views/utils/file_utils_2_3_tests.py

```diff
@@ -11,13 +15,13 @@
 
 TSV_DATA = b'Family ID	Individual ID	Notes\n\
 "1"	"NA19675"	"An affected individual, additional metadata"\n\
-"1"	"NA19678"	""'
+"1"	"NA19678"	"None"'
```

Comment:
For `.xlsx` format `""` will get a parsed value of `None` rather than an empty string `''`. It need a special handling for the parsed data. I will try to update it.